downstream/modules/platform/ref-controller-database-settings.adoc

```diff
@@ -25,6 +25,13 @@ If you are compiling Postgres against OpenSSL 3.2, your system regresses to remo
 You must restart the database server after changing the value for `shared_buffers`.
 ====
 
+Warning
```

Comment:
I think this should be [WARNING]

Reply:
This was supposed to be a note actually. Jameria and I had a longer conversation about it and it wasn't catastrophic to the user, so we landed on NOTE, so I'm not sure why it didn't follow this copy.